stellargraph/layer/gcn_lstm.py

```diff
@@ -293,6 +296,8 @@ def __init__(
                 )
             )
 
+        self._layers.append(Permute((2, 1)))
```

Comment:
maybe a short comment here would be helpful
```suggestion
        # put time-sequence dimension last for LSTM layers
        self._layers.append(Permute((2, 1)))
```